lms/djangoapps/certificates/management/commands/cert_whitelist.py

```diff
@@ -114,7 +113,6 @@ def update_user_whitelist(username, add=True):
         )
         for whitelisted in whitelist:
             username = whitelisted.user.username
-            username = username.encode('utf-8')
```

Comment:
> The username field is a unicode
string already so it does not need to be encoded
again, and was causing an error on print.

Just to check, did we test this on stage w/ users w/ unicode names?
They're normally not allowed, so it would have had to be tested explicitly.

Reply:
yeah, we went and changed usernames (ours) to include unicode characters and ran the command to confirm it worked.